### ISSUE TITLE
Enable / Disable Syphon Server

### ISSUE THREAD

**Issue by hossimo** [Syphon/Quartz-Composer#2]
It seems if you Disable then re-enable a Syphon server it does not displayed as a syphon source until you restart the composition.

Plugin V4
Version 4.6 (151)
10.9.5

**Comment by vade** [comment]
Interesting. Thanks for the report. Ill take a peek in a bit. Please remind me if I don't get back to you shortly, I'm juggling a few things of late :)

**Comment by hossimo** [comment]
No problem, I was going to test on my 10.10 machine but then noticed I didn't have QC installed.

**Comment by vade** [comment]
Hrm. I cant recreate on 10.10 with the latest Syphon QC plugin. 

Are you running the latest?

I can disable the plugin, my Server doesnt update (in recorder) if I make changes to the scene in QC.

if I re-enable Syphon Server patch, Recorder or another client instantly picks back up.

**Comment by hossimo** [comment]
/Library/Graphics/Quartz Composer Plug-Ins shows
Syphon Quartz Composer Plug-In 4, Copyright Anton Marini & Tom Butterworth, 2010-2014

I did reinstall v4 yesterday before testing again but I did not restart. This morning on my 10.10.3 and 10.9.5 machines I am seeing something different using the [same QC file](https://drive.google.com/a/downrighttech.com/file/d/0B4xg0DZ8ad4geGx3M1I5UE1GZ0k/view?usp=sharing), now the enable does not seem to do anything. 

I just installed QC on my 10.10 machine and restarted my 10.9 machine last night so that might have something to do with it.
